.github/workflows/codespell.yml

```diff
@@ -0,0 +1,18 @@
+on:
+    push:
+        branches: [master]
+    pull_request:
+
+name: Spell Check
+ 
+jobs:
+    codespell:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: codespell-project/actions-codespell@master
+              with:
+                  check_filenames: true
+                  only_warn: 1
+                  ignore_words_list: cas,objext,ba,probleme,formend,compiletime,chage,hve,files,ans,inout,testin,files\',creat
```

Comment:
I think it might be better to use `ignore_words_file` as this might grow to a (very) long list.

Reply:
Should now be implemented. Had to do some extra code to do the same thing for the files to skip.